nuget/bootstrap.nuspec

```diff
@@ -18,7 +18,7 @@
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <dependencies>
       <dependency id="jQuery" version="[3.0.0,4)" />
-      <dependency id="popper.js" version="[1.16.0,2)" />
+      <dependency id="popper.js" version="[1.16.1-lts,2)" />
```

Comment:
Now that I think about it, maybe this allows for non-LTS versions.

@supergibbs what do you think?

Reply:
the NuGet package doesn't ship with a deprecation warning, so it doesn't really make any difference.

Reply:
Adding a `-lts` to semver [designates it as pre-release](https://semver.org/#spec-item-9) so any platform that implements it correctly will think `1.16.1-lts` is before `1.16.1`. You can see that on [nuget.org](https://www.nuget.org/packages/popper.js/). This also [applies to npm](https://semver.npmjs.com/); installing `^1.16` would get you `1.16.1` not `1.16.1-lts`. It would be better to release a `1.16.2` and mention in the release notes it's LTS. You can also deprecate it on nuget.org to let users know it's no longer maintained.

https://docs.microsoft.com/en-us/nuget/nuget-org/deprecate-packages

Reply:
I don't plan to deprecate the NuGet package, I didn't want to deprecate the npm either, but it's the only way to warn users about a package rename.

The NuGet support for Popper ended with 1.x so there's not a new namespace to point the users to.

Personally I would keep the NuGet dependency at 1.16.1 and update only the npm dependency.

Reply:
Ya, if there is no functional differences I’d just leave the dependency as 1.16.1

Another side note, LTS is often on a minor version, in this case 1.16, meaning no new features (aka 1.17) just security patches going forward. Then 1.16.2 would then be lts too if there was some critical reason to release it. 👍🏼